core/backend/src/test/integration/Checkpoints.test.ts

```diff
@@ -16,12 +16,9 @@ import { HubUtility } from "./HubUtility";
 
 // FIXME: Disabled because V2 checkpoints are not in QA yet...

```

Comment:
Tangential to this PR but I think this comment is no longer accurate.

Reply:
I'll verify with @wgoehrig but I think that's still accurate.  v2 checkpoints are in DEV currently, hopefully QA by EOW.

Reply:
yeah, @calebmshafer, that's right.